core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -695,7 +734,7 @@ public static function findPublishedRegularWithoutGuestsByIds($arrIds, array $ar
 
 		if (System::getContainer()->get('contao.security.token_checker')->hasFrontendUser())
 		{
-			$arrColumns[] = "$t.guests=''";
+			$arrColumns[] = "$t.guests=''"; # FIXME
```

Comment:
@contao/developers Any idea how to fix this? We would have to call `->loadDetails()` to know if the page or one of its parent pages is visible to guests only.

Reply:
Fixed in 4de28acf3526481443d8ee6960eb1e0611f01e2f.